portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,102 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Hello, my name is Sydney Beasley</title>
+    <title>Sydney Beasley Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body style="background-color:rgb(233, 206, 255);">
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>
+        <intro-title> Hello, my name is Sydney!</intro-title>
+      </h1>
+      <br>
```

Comment:
I suggest not using a `br` here. If you need extra space between the `h1` and the button, you can add margin either to the bottom of `h1` or to the top of your button.